### DISCUSSION TITLE
What is the API used to get hafs and warsh qpc fonts ?

### DISCUSSION THREAD

**Discussion by MahmoudMasab** [quran/discussions#2] [Q&A]
What is the API used to get hafs and warsh qpc fonts of 604 pages like mushaf-v2 ( QCF2001.ttf ......  QCF2604.ttf )

**Comment by yoursjerry** [comment]
> What is the API used to get hafs and warsh qpc fonts of 604 pages like mushaf-v2 ( QCF2001.ttf ...... QCF2604.ttf )

Find fonts here https://github.com/mustafa0x/qpc-fonts/
